soc/arm/st_stm32/stm32h7/Kconfig.soc

```diff
@@ -7,6 +7,11 @@ choice
 	prompt "STM32H7x MCU Selection"
 	depends on SOC_SERIES_STM32H7X
 
+config SOC_STM32H743XX
+	bool "STM32H743XX"
+	select CPU_CORTEX_M7
```

Comment:
I just want to check with you regarding the selection of CPU_CORTEX_M7 here.
SOC_STM32H747XX, below, does not select the Cortex-M option; that's because it is selected by the Board target (may be an M4 or M7 depending on the target). Is it the same case for SOC_STM32H743XX, or is this only an M7 SoC? If not, I suggest we move the CPU selection at the board level.

Reply:
@ioannisg This H7 reference is single core

Reply:
747XX does not select the Cortex-M option because it can either be M4 or M7. 743XX are not dual core SoC, so they're always M7.